broker/src/main/java/io/moquette/broker/scheduler/ScheduledExpirationService.java

```diff
@@ -25,13 +25,14 @@ public class ScheduledExpirationService<T extends Expirable> {
     private final ScheduledFuture<?> expiredEntityTask;
     private final Clock clock;
     private final Consumer<T> action;
+    ScheduledExecutorService actionsExecutor;
```

Comment:
Given that `actionsExecutor` is assigned only once in the constructor, this can be 

```suggestion
    private final ScheduledExecutorService actionsExecutor;
```

Reply:
Oops, fixed :)